Sources/StringsLintFramework/Models/LocalizedString.swift

```diff
@@ -12,6 +12,7 @@ public struct LocalizedString: CustomStringConvertible, Hashable {
     let location: Location
     var placeholders: [String] = []
     var comment: String? = nil
+    var value: [String]? = nil
```

Comment:
Why is the value an array?

Reply:
I was assuming we need to make sure every possible string in `StringsDict` also don't exceed the length?

Reply:
Ah yes, good call. I missed that.